app/views/docs/installation.phtml

```diff
@@ -4,6 +4,8 @@
 
 <p>Appwrite was designed to run well on both small and large deployment. The minimum requirements to run Appwrite is as little as <b>1 CPU core</b> and <b>2GB of RAM</b>, and an operating system that supports Docker.</p>
 
+<p>As of version 0.14, Appwrite requires <a href="https://www.docker.com/blog/announcing-compose-v2-general-availability/" target="_blank" rel="noopener">Docker Compose Version 2</a>. To install Appwrite, make sure your Docker installation is updated to support Composer V2.</p>
```

Comment:
The link here provides good information on compose V1 vs V2, but I'm worried this info would be too confusing to users new to Docker. 

Is it also worth it to provide a direct link to how to [install](https://docs.docker.com/compose/install/)?

Also, by default, the new compose plugin would be executed with `docker compose` but much of the docs use `docker-compose`. One would need to set up some alias or something to make `docker-compose` execute `docker compose`. Without mentioning that or updating the rest of the docs to use `docker compose`, users might get confused.

Reply:
Yeah this link -> https://www.docker.com/blog/announcing-compose-v2-general-availability/

Belongs in the migration guide. The installation page from Docker should be in installation.

I think I do need to update everything to `docker compose` since we have linux users and most of them (I'm guessing) do not use desktop.